shell/platform/windows/window_state.h

```diff
@@ -19,13 +19,6 @@ struct FlutterWindowsEngine;
 struct FlutterWindowsView;
 }  // namespace flutter
 
-// Wrapper to distinguish the view controller ref from the view ref given out
-// in the C API.
-struct FlutterDesktopViewControllerState {
-  // The view that backs this state object.
-  std::unique_ptr<flutter::FlutterWindowsView> view;
-};
```

Comment:
This is a legacy struct that is intended to be phased out: https://github.com/flutter/engine/blob/e3e59ba2e09387d3d82aca5a27f900a2546a4713/shell/platform/windows/window_state.h#L12-L15